Add new post about removing the FIPS certified check from enable_fips_mode

ggbecker opened the pull request.

jan-cerny commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,40 @@
+system has the FIPS mode enabled in the Operating System (OS). This means that cryptographic modules are
+running in a mode that only NIST approved algorithms, ciphers and everything related
+to cryptography are allowed to be used. BUT, it doesn't necessarily mean that
+it is a NIST certified OS (cryptographic modules) that meets the
```
This sentence is weird, mainly due to parenthesis.

For example:

But, it doesn't mean that the system's cryptography modules have met all FIPS 140-2 requirements and it doesn't mean that the system has received FIPS 140-2 certification by NIST.

jan-cerny commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,40 @@
+to cryptography are allowed to be used. BUT, it doesn't necessarily mean that
+it is a NIST certified OS (cryptographic modules) that meets the
+FIPS 140-2 requirements. For that to happen, the OS has to undergo
+through NIST's FIPS 140-2 certification process.
```
Idea: Maybe, we could rework the whole paragraph in the following way or add a summary sentence in the following way:

If the rule passes, it means that ..., If the rule passes, it doesn't mean that ...,

jan-cerny commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,40 @@
+running in a mode that only NIST approved algorithms, ciphers and everything related
+to cryptography are allowed to be used. BUT, it doesn't necessarily mean that
+it is a NIST certified OS (cryptographic modules) that meets the
+FIPS 140-2 requirements. For that to happen, the OS has to undergo
```
You can add something that emphasizes your statement, for example: Therefore, the check can pass also on non-certified systems.

jan-cerny commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,40 @@
+would only pass if the OS is included in this
+[list](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/oval/shared.xml).
+
+The community has already tried to go in this direction in the past but it was
+rejected. For more details you can check these links:
+[PR#4920](https://github.com/ComplianceAsCode/content/pull/4920) and [Issue#4917](https://github.com/ComplianceAsCode/content/issues/4917).
```
I don't find this interesting.

ggbecker replied:
I agree, it can cause more controversies

ggbecker replied:
I'll keep it commented to keep the links saved

jan-cerny commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,40 @@
+[installed_OS_is_FIPS_certified](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml)
+that contains the list of FIPS certified Operating Systems (once again,
+certified cryptographic modules). So this means the rule `enable_fips_mode`
+would only pass if the OS is included in this
```
```suggestion
had only pass if the OS is included in this
```

ggbecker replied:
I changed to `had only passed if the OS was included in this`. Crazy verb tenses.

jan-cerny reviewed (approved):
It's amazing and I have only minor suggestions.

ggbecker commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,40 @@
+project](https://github.com/ComplianceAsCode/content/) contains the rule [enable_fips_mode](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/integrity/fips/enable_fips_mode/rule.yml) that checks if a
+system has the FIPS mode enabled in the Operating System (OS). This means that cryptographic modules are
+running in a mode that only NIST approved algorithms, ciphers and everything related
+to cryptography are allowed to be used. BUT, it doesn't necessarily mean that
```
I wanted to scream here.

ggbecker commented:
@jan-cerny I tried to address the feedback the way I could. Please have another look. I'm attaching a rendered image for better reading.

![Screenshot from 2022-03-10 12-55-45](https://user-images.githubusercontent.com/18730394/157657006-532e6b9c-0798-4c25-a043-c460b2140dd6.png)

jan-cerny reviewed (approved):
Great!

ggbecker commented on _posts/2022-03-09-removing-os-fips-verification.md:
```diff
@@ -0,0 +1,38 @@
+systems database](https://csrc.nist.gov/projects/cryptographic-module-validation-program/validated-modules/search).
+
+This change was done in
+[PR#8255](https://github.com/ComplianceAsCode/content/pull/8255) and will be
+available in the next release 0.1.61.
```
```suggestion
[PR#8255](https://github.com/ComplianceAsCode/content/pull/8255) and is
available since release 0.1.61.
```

ggbecker reviewed (commented):
before merging, make sure the date is correct.

jan-cerny commented:
@ggbecker What blocks us from making this ready for review and from merging it?

ggbecker commented:
> @ggbecker What blocks us from making this ready for review and from merging it?

I've updated the date of the post and moved it to ready to review. I believe we can merge it
